420 classic dyalog mode (#958)
Dyalog/documentation: windows-ui-guide/docs/editor.md

## Window Management (Standard)

> In Standard mode, which is selected with either *Debugger at the bottom* or *Debugger on the left* from the *Layout* menu, the Editor is a Multiple Document Interface (MDI) window that may be a stand-alone window, or be docked in the Session window. Each of the objects being edited is displayed in a separate sub-window. Individual edit windows are managed using standard MDI facilities.

Reviewer: ```suggestion
In Standard mode (either **Debugger at the bottom** or **Debugger on the left** are selected from the **Layout** menu), the Editor is a Multiple Document Interface (MDI) window that can be either a stand-alone window or docked in the Session window. Each of the objects being edited is displayed in a separate sub-window. Individual Edit windows are managed using standard MDI facilities.
```

## Window Management (Classic Dyalog mode)

> In Classic Dyalog mode, which is selected with *Classic* from the *Layout* menu, each Edit window is a top-level window created as a child of the Session window. This means that normally Edit windows appear on top of the Session.

Reviewer: ```suggestion
In Classic Dyalog mode (**Classic** is selected from the **Layout** menu), each Edit window is a top-level window created as a child of the Session window. Edit windows usually appear on top of the Session window.
```

> The first Edit window is created at the position specified by the **edit_first_y** and **edit_first_x** parameters which  are specified in terms of the size of a character in the current font relative to the top-left corner of the screen.

Reviewer: ```suggestion
The first Edit window is created at the position specified by the edit_first_y and edit_first_x configuration parameters; these are specified in terms of the size of a character in the current font relative to the top-left corner of the screen.
```
